Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -940,10 +952,9 @@
   - UNKNOWN CREATURES
   - VAMPIRES
   - VELOCIRAPTORS
-  - VIRUSES
+  # - VIRUSES not real yet
   - WEREWOLVES
   - WIZARDS
-  - XENOS
```

Comment:
xenos are infact real, maybe XENOMORPHS if current-day xenophobia is the concern

Reply:
werent they removed for copyright? it wasnt for xenophobia lol

Reply:
Il add them back as XENOMORPHS then

Reply:
good call